webapp/public/javascripts/controllers.js

```diff
@@ -60,4 +63,18 @@ lunchrControllers.controller('RegisterController', ['$scope', '$http', '$state',
                     $scope.errorMessages = data;
                 })
         }
+    }]);
+
+lunchrControllers.controller('UserMatchingController', ['$state', 'socket',
+    function ($state, socket) {
+        socket.emit('match');
```

Comment:
I'll look into it. Might be worth doing that in another PR if we deem it necessary.